kytos/cli/commands/napps/api.py

```diff
@@ -146,10 +148,11 @@ def install_napps(cls, napps):
                             cls.install_napps(napp_dependencies)
                     else:
                         LOG.warning('  Napp already enabled.')
-                except KytosException:
-                    continue
-            else:
-                LOG.warning('  Napp already enabled.')
+                else:
+                    LOG.warning('  Napp already enabled.')
```

Comment:
I think the last message should be 'already installed' instead of 'already enabled'.

Reply:
You are correct. I will modify it.